Makefile

```diff
@@ -51,10 +51,10 @@ image_builder=
 build:
 	./hack/go-build.sh --image_registry ${image_registry} --image_name ${image_name} --image_tag ${image_tag} --image_builder ${image_builder}
 
-.PHONY: install
+.PHONY: deploy-operator-on-ocp
 image=
-install:
-	./hack/install-operator.sh $(image)
+deploy-operator-on-ocp:
+	./hack/deploy-operator-on-ocp.sh $(image)
```

Comment:
is this action supported on ocp 3.11 and 4.x?

Reply:
Yes, it is. I've just checked with Openshift 3.11.